device-types/Synology/RS2416RP+.yaml

```diff
@@ -0,0 +1,37 @@
+---
+manufacturer: Synology
+model: RS2416RP+
+slug: synology-rs2416rp-plus
+part_number: RS2416RP+
+u_height: 2
+is_full_depth: true
+weight: 13.24
+weight_unit: kg
+airflow: front-to-rear
+front_image: true
+rear_image: true
+comments: '[Synology RS2416RP+ Data Sheet](https://global.synologydownload.com/download/Document/Hardware/DataSheet/RackStation/16-year/RS2416RP+/enu/Synology_RS2416_Plus_RS2416RP_Plus_Data_Sheet_enu.pdf)'
+console-ports:
+  - name: Console
+    type: de-9
+    mgmt_only: true
+power-ports:
+  - name: PSU 1
+    type: iec-60320-c14
+    maximum_draw: 120
+    allocated_draw: 56
+  - name: PSU 2
+    type: iec-60320-c14
+    maximum_draw: 120
+    allocated_draw: 56
```

Comment:
These PSUs are modular, so they should be modelled under `module-bays` and the associated PSUs themselves should be modelled in the module-types folder if they do not already exist